README.md

```diff
@@ -72,21 +72,21 @@ Full examples of usage scripts are following:
 # for prometheus pushgateway
 export PROM_ADDRESS="localhost:9091"
 export PROM_JOB=pushgateway
-./bblfsh-performance end-to-end --language=go --commit=3d9682b --extension=".go" --storage="prom" /var/testdata/benchmarks
+./bblfsh-performance end-to-end --language=go --commit=3d9682b --filter-prefix="bench_" --exclude-substrings=".legacy",".native",".uast" --storage="prom" /var/testdata/benchmarks
```

Comment:
It's a bit unusual syntax: `".legacy",".native",".uast"`. Have you tested it?
I haven't used it that way, thus wondering if it's correct to do so.

Reply:
tested